init.sh

```diff
@@ -162,6 +161,30 @@ printMessage () {
     esac
 }
 
+installMustacheEngine () {
```

Comment:
Could we prevent needing this dependency? Having templates makes the solution very complex vs individual files for each use case. Another viable solution to keep things simple may be forking catalyst-owner into catalyst-owner-readonly repo to keep it as simple as possible. 

I think people won't benefit from this templates and conditionals in their operations. And contributors to this repository wont be benefited from the added complexity